__tests__/servertesting.test.js

```diff
@@ -576,3 +576,29 @@ describe("/api/articels/?topic", () => {
       });
   });
 });
+describe("/api/articles/:article_id?comment_count", () => {
+  it("200: Should return article object with comment_count attached with correct value", () => {
+    return request(app)
+      .get("/api/articles/9/?comment_count")
```

Comment:
This shouldn't be a query, all articles should have a comment count attached to them. So the endpoint would still be /api/articles/9